packages/riipen-ui/src/components/RadioGroup.test.jsx

```diff
@@ -0,0 +1,153 @@
+      expect(childrenNodes.at(1).props().label).toEqual("two");
+    });
+  });
+
+  describe("error prop", () => {
+    it("displays error", () => {
+      const error = <span>an error!</span>;
+
+      const wrapper = mount(<RadioGroup error={error} />);
+
+      expect(wrapper.find("fieldset").contains(error)).toEqual(true);
+    });
+  });
+
+  describe("hint prop", () => {
+    it("renders InputLabel when hint is provided", () => {
+      const hint = "it's a hint";
+
+      const wrapper = mount(<RadioGroup hint={hint} />);
+
+      expect(wrapper.find("InputLabel")).toHaveLength(1);
+    });
+
+    it("passes given hint to InputLabel", () => {
+      const hint = "it's a hint";
+
+      const wrapper = mount(<RadioGroup hint={hint} />);
+
+      expect(wrapper.find("InputLabel").props().hint).toEqual(hint);
+    });
+  });
+
+  describe("label prop", () => {
+    it("renders InputLabel when label is provided", () => {
+      const label = "it's a label";
+
+      const wrapper = mount(<RadioGroup label={label} />);
+
+      expect(wrapper.find("InputLabel")).toHaveLength(1);
+    });
+
+    it("passes given label to InputLabel as children", () => {
+      const label = "it's a label";
+
+      const wrapper = mount(<RadioGroup label={label} />);
+
+      expect(wrapper.find("InputLabel").props().children).toEqual(label);
+    });
+
+    it("does not render InputLabel when both label and hint are not provided", () => {
+      const wrapper = mount(<RadioGroup />);
+
+      expect(wrapper.find("InputLabel")).toHaveLength(0);
+    });
+  });
+
+  describe("onChange prop", () => {
+    it("sets onChange prop of children nodes", () => {
+      const child = <Radio label="one" readOnly />;
+
+      const wrapper = mount(<RadioGroup>{child}</RadioGroup>);
+
+      expect(wrapper.find("Radio").props().onChange).toBeDefined();
+    });
+
+    it("invokes handler when child Radio is changed", () => {
+      const child = <Radio label="one" readOnly />;
+      const onChange = jest.fn();
+
+      const wrapper = mount(
+        <RadioGroup onChange={onChange}>{child}</RadioGroup>
+      );
+
+      wrapper
+        .find("Radio")
+        .find("input")
+        .simulate("change");
+
+      expect(onChange).toHaveBeenCalledTimes(1);
+    });
+  });
+
+  describe("required prop", () => {
+    it("sets required prop of InputLabel to true when required is true", () => {
+      const label = "it's a label";
+      const required = true;
+
+      const wrapper = mount(<RadioGroup label={label} required={required} />);
+
+      expect(wrapper.find("InputLabel").props().required).toEqual(required);
+    });
+
+    it("sets required prop of InputLabel to false when required is false", () => {
+      const label = "it's a label";
+      const required = false;
+
+      const wrapper = mount(<RadioGroup label={label} required={required} />);
+
+      expect(wrapper.find("InputLabel").props().required).toEqual(required);
+    });
```

Comment:
I don't think you need both of these. You could just have a test "passes required prop to InputLabel" or something.